scripts/build_reproducibility_index

```diff
@@ -13,7 +13,8 @@ source "$SCRIPTS_DIR/common"
 # List of artifacts that are expected to be reproducibly built.
 readonly REPRODUCIBLE_ARTIFACTS=(
   ./target/wasm32-unknown-unknown/release/*.wasm
-  ./target/x86_64-unknown-linux-musl/release/oak_loader
+  # TODO(#725): Include this line when `./scripts/build_server -s rust` actually builds the binary.
+  # ./target/x86_64-unknown-linux-musl/release/oak_loader
```

Comment:
@ipetr0v BTW I am not sure why this is not the case currently? Is it intended?

Reply:
Ah I think https://github.com/project-oak/oak/pull/932 was the cause.

Reply:
Yes, currently even the Rust Loader is built by Bazel